ebpfdiscoverysrv/src/main.cpp

```diff
@@ -182,6 +183,7 @@ int main(int argc, char** argv) {
 	perf_buffer* logBuf{bpflogging::setupLogging(logPerfBufFd)};
 	if (logBuf == nullptr) {
 		LOG_CRITICAL("Could not open perf buffer for Discovery BPF logging: {}.", std::strerror(-errno));
```

Comment:
```suggestion
		LOG_CRITICAL("Could not open perf buffer for Discovery BPF logging: {}.", std::strerror(errno));
```
This suggestion should fix the error message being "Unknown error"